package.json

```diff
@@ -6,7 +6,7 @@
     "node": "14 || 16"
   },
   "scripts": {
-    "dev": "BACKEND_SECRET=secret concurrently \"yarn start\" \"yarn start-backend\"",
+    "dev": "set -a && source ./.env && set +a && BACKEND_SECRET=secret concurrently \"yarn start\" \"yarn start-backend\"",
```

Comment:
Hm, can you please move the `BACKEND_SECRET` to the `.env` file as well?

I don't like the messy set+source+set, it's a shame backstage cli doesn't support dotenv... Can you please instead try installing `env-cmd` as a dev dependency to root of all workspaces and extend the `start-backend` script with it? We shouldn't need env variables on the frontend. https://github.com/toddbluhm/env-cmd

Reply:
Moved to another PR.